### DISCUSSION TITLE
Borders Weird.

### DISCUSSION THREAD

**Discussion by ThePunkSky** [i3/i3#5603] [Q&A]
My borders are being funky. I run picom(no fork) and i3(no fork) and I get this pink bar on when I only have one thing open (image 1). And when I have multiple terms open the round corners (radius 15 in picom.conf) it cuts into the borders(image 2). My question is two fold: what causes the pink bar and how do I stop it. And how can I just disable borders altogether?
![bordersWeird2](https://github.com/i3/i3/assets/102258343/db83c010-b5ab-4fd4-b2f1-65e786fddad0)
![bordersWeird3](https://github.com/i3/i3/assets/102258343/38d9def5-b57e-4b11-8039-881ead3d3dde)

**Comment by orestisfl** [comment]
Try to include your config files for i3 and picom, their versions, driver information and distro.

**Comment by okraits** [comment]
It is documented in the userguide how to disable borders:

https://i3wm.org/docs/userguide.html#default_border

`default_border none`
`default_floating_border none`
